server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -1171,147 +1173,205 @@ private String validateConfigurationValue(final String name, String value, final
         if (type.equals(Integer.class) && NetworkModel.MACIdentifier.key().equalsIgnoreCase(name)) {
             try {
                 final int val = Integer.parseInt(value);
-                //The value need to be between 0 to 255 because the mac generation needs a value of 8 bit
+                //The value needs to be between 0 to 255 because the mac generation needs a value of 8 bit
                 //0 value is considered as disable.
                 if(val < 0 || val > 255){
-                    throw new InvalidParameterValueException(name+" value should be between 0 and 255. 0 value will disable this feature");
+                    throw new InvalidParameterValueException(String.format("[%s] value should be between 0 and 255. 0 value will disable this feature.", name));
                 }
             } catch (final NumberFormatException e) {
-                s_logger.error("There was an error trying to parse the integer value for:" + name);
-                throw new InvalidParameterValueException("There was an error trying to parse the integer value for:" + name);
+                s_logger.error(String.format("There was an error trying to parse the integer value for: [%s].", name));
```

Comment:
let's extract this format string in a constant. it is used in four places.
Also let's construct the message one time for both the log and the exception below.